MFVNeutralino/plugins/Printer.cc

```diff
@@ -163,20 +163,20 @@ void MFVPrinter::analyze(const edm::Event& event, const edm::EventSetup& setup)
     for (int i = 0; i < 3; ++i)
       printf("%u ", mevent->nbtags(i));
     printf("\n");
-    printf("nmu (any,sel): ");
-    for (int i = 0; i < 2; ++i)
-      printf("%i ", mevent->nmu(i));
-    printf("  nel (any,sel): ");
-    for (int i = 0; i < 2; ++i)
-      printf("%i ", mevent->nel(i));
-    printf("  nlep (any,sel): ");
-    for (int i = 0; i < 2; ++i)
-      printf("%i ", mevent->nlep(i));
-    size_t nl = mevent->nlep();
-    die_if_not(nl == mevent->lep_qpt.size() && nl == mevent->lep_eta.size() && nl == mevent->lep_phi.size() && nl == mevent->lep_dxy.size() && nl == mevent->lep_dz.size() && nl == mevent->lep_iso.size(), "lep vectors not same size");
-    printf("raw lep info (nl: %lu):\n", nl);
-    for (size_t i = 0; i < nl; ++i)
-      printf("id: 0x%08x   pt: %11.3g   eta: %11.3g   phi: %11.3g   dxypv: %11.3g   dxybs: %11.3g   dz: %11.3g   iso: %11.3g\n", mevent->lep_id_[i], mevent->lep_qpt[i], mevent->lep_eta[i], mevent->lep_phi[i], mevent->lep_dxy[i], mevent->lep_dxybs[i], mevent->lep_dz[i], mevent->lep_iso[i]);
+    // printf("nmu (any,sel): ");
+    // for (int i = 0; i < 2; ++i)
+    //   printf("%i ", mevent->nmu(i));
+    // printf("  nel (any,sel): ");
+    // for (int i = 0; i < 2; ++i)
+    //   printf("%i ", mevent->nel(i));
+    // printf("  nlep (any,sel): ");
+    // for (int i = 0; i < 2; ++i)
+    //   printf("%i ", mevent->nlep(i));
+    // size_t nl = mevent->nlep();
+    // die_if_not(nl == mevent->lep_qpt.size() && nl == mevent->lep_eta.size() && nl == mevent->lep_phi.size() && nl == mevent->lep_dxy.size() && nl == mevent->lep_dz.size() && nl == mevent->lep_iso.size(), "lep vectors not same size");
+    // printf("raw lep info (nl: %lu):\n", nl);
+    // for (size_t i = 0; i < nl; ++i)
+    //   printf("id: 0x%08x   pt: %11.3g   eta: %11.3g   phi: %11.3g   dxypv: %11.3g   dxybs: %11.3g   dz: %11.3g   iso: %11.3g\n", mevent->lep_id_[i], mevent->lep_qpt[i], mevent->lep_eta[i], mevent->lep_phi[i], mevent->lep_dxy[i], mevent->lep_dxybs[i], mevent->lep_dz[i], mevent->lep_iso[i]);
 
```

Comment:
If we might want these printouts in the future, do we want this commented out? Or do we want to put it into a verbose option?